packages/suite-desktop/src-electron/modules/shortcuts.ts

```diff
@@ -23,9 +22,11 @@ const init = ({ mainWindow, src }: Dependencies) => {
         mainWindow.loadURL(src);
     });
 
-    app.on('before-quit', () => {
-        electronLocalshortcut.unregisterAll(mainWindow);
-    });
+    // todo: calling electronLocalshortcut.unregisterAll(mainWindow) has unpleasant side-effect
```

Comment:
I believe that all local shortcuts should be unregistered at that point already by `electron-localshortcut` itsel

Reply:
removed commented code in [12baf63](https://github.com/trezor/trezor-suite/pull/4931/commits/12baf635bfab1e99ce895baf5c4b9f3d968e627d)